src/page.rs

```diff
@@ -96,19 +97,22 @@ fn get_links(response: &Response) -> Result<(Option<Url>, Option<Url>)> {
     let mut prev = None;
     let mut next = None;
 
-    if let Some(link_header) = response.headers().get::<Link>() {
-        for value in link_header.values() {
-            if let Some(relations) = value.rel() {
+    let link_header = response.headers().get_all(LINK);
+    for value in &link_header {
+        let val = value.to_str().unwrap();
+        let raw = std::convert::From::from(val);
```

Comment:
Is there a reason you're calling `From::from` instead of `let raw = val.into();`?